src/handleAction.js

```diff
@@ -1,19 +1,18 @@
 import isFunction from 'lodash/isFunction';
 import identity from 'lodash/identity';
 import isNil from 'lodash/isNil';
-import isSymbol from 'lodash/isSymbol';
+import includes from 'lodash/includes';
+import { ACTION_TYPE_DELIMITER } from './combineActions';
 
-export default function handleAction(type, reducers, defaultState) {
-  const typeValue = isSymbol(type)
-    ? type
-    : type.toString();
+export default function handleAction(actionType, reducers, defaultState) {
+  const actionTypes = actionType.toString().split(ACTION_TYPE_DELIMITER);
```

Comment:
For performance we could do a check before this if `actionType` includes `ACTION_TYPE_DELIMITER`, otherwise skip this transformation, because I think the majority of `actionTypes` will be non-combined action types.